### ISSUE TITLE
Ideally this should be made available via Maven

### ISSUE THREAD

**Issue by alexpetlenko** [apanimesh061/VaderSentimentJava#3]
It would be really good if releases were deployed to maven so this project can be pulled as a simple dependency.

**Comment by apanimesh061** [comment]
@alexpetlenko That is a good idea. Let me see how to do that.

**Comment by apanimesh061** [comment]
@alexpetlenko I have uploaded the jar to Nexus repository. You can get download the dependency by adding the following to you `pom.xml`:

```
<dependency>
  <groupId>com.github.apanimesh061</groupId>
  <artifactId>vader-sentiment-analyzer</artifactId>
  <version>1.0</version>
</dependency>
```

Path to jar: [https://oss.sonatype.org/service/local/repositories/releases/content/com/github/apanimesh061/vader-sentiment-analyzer/1.0/vader-sentiment-analyzer-1.0.jar](https://oss.sonatype.org/service/local/repositories/releases/content/com/github/apanimesh061/vader-sentiment-analyzer/1.0/vader-sentiment-analyzer-1.0.jar)

**Comment by alexpetlenko** [comment]
Great stuff - thank you.

**Comment by nngo** [comment]
@apanimesh061 is the release you published to maven central version 1.0 or 1.0.1? Also, when do you plan on releasing a 1.0.2 or 1.1 version?
